doc/PostProcessing.md

```diff
@@ -420,6 +424,62 @@ The `"name"` and `"varexp"` are the only compulsory arguments, others can be omi
+
+```json
+{
+         ...
+         "plots": [
+          {
+            "name": "trend_cycle_duration_ntf_corr",
+            "title": "cycle duration: ns/TF;time;cycle duration [ns/TimeFrames]",
+            "legend": { "enabled": true, "x1": 0.70, "y1": 0.70, "x2": 0.93, "y2": 0.90, "nColumns": 1 },
+            "graphs": [
+              {
+                "title": "cycle duration [ns]",
+                "varexp": "CycleDuration.entries:time",
+                "selection": "",
+                "option": "*LP",
+                "style": { "lineColor": 38, "markerColor": 4, "markerStyle": 20, "lineWidth": 2 }
+              },
+              {
+                "title": "cycle duration [TimeFrames]",
+                "varexp": "CycleDurationNTF.entries:time",
+                "selection": "",
+                "option": "*LP",
+                "style": { "lineColor": 8, "markerColor": 3, "markerStyle": 30, "lineWidth": 2 }
+              },
+              {
+                "title": "cycle duration: ns/TF;time;cycle duration [ns/TimeFrames]",
+                "varexp": "CycleDuration.entries/CycleDurationNTF.entries:time",
+                "selection": "",
+                "option": "*LP",
+                "style": { "lineColor": 31, "markerColor": 30, "markerStyle": 23, "lineWidth": 2 }
+              }
+            ]
+          }
+        ],
+	...
+}
+```
+
+The specific options are:
+
+* `"legend"` ~ allows to specify precise location and shape of legend, as well as disable it completely
+  * `"x1"`, `"y1"`, `"x2"`, `"y2"` ~ manual coordinates of the legend box, all need to be present for custom legend placement to take account
+  * `"nColumns"` ~ number of columns in the legend
+* `"style"` ~ allows to change the style of the line and marker
+  * `"lineWidth"` ~ width of the line, integer as defined by ROOT [TAttLine class](https://root.cern/doc/master/classTAttLine.html)
+  * `"lineStyle"` ~ style of the line, integer as defined by ROOT [TAttLine class](https://root.cern/doc/master/classTAttLine.html)
+  * `"lineColor"` ~ color of the line, an integer as defined by ROOT [TColor class](https://root.cern/doc/master/classTColor.html)
+  * `"markerColor"` ~ color of the marker, integer as defined by ROOT [TAttMarker class](https://root.cern/doc/master/classTAttMarker.html)
+  * `"markerStyle"` ~ style of the marker, integer as defined by ROOT [TAttMarker class](https://root.cern/doc/master/classTAttMarker.html)
+  * `"markerSize"` ~ size of the marker, float as defined by ROOT [TAttMarker class](https://root.cern/doc/master/classTAttMarker.html)
+  * `"fillColor"` ~ color of the fill, an integer as defined by ROOT [TColor class](https://root.cern/doc/master/classTColor.html)
+  * `"fillStyle"` ~ style of the fill, integer as defined by ROOT [TAttFill class](https://root.cern/doc/master/classTAttFill.html)
+
+WARNING: Any style parameters specified will override colliding parameters in `"option"`.
```

Comment:
```suggestion
WARNING: Any style parameters specified will override colliding parameters in `"option"`.
However, lines, fills and markers still have to be enabled in `"option"` to appear (e.g. with `"option" : "LPF"`).
```
While playing with the feature, I discovered that indeed the style parameters override the colliding settings in `"option"`, but markers, fills and lines still have to be enabled in `"option"` (e.g. with "LP"), using just `"style"` is not enough to make them appear. This is fine, but we could add a sentence mentioning that.